KW is a PE not just a KDAC. (#275)
kubewarden/docs: docs/introduction.md

keywords: [kubewarden, cncf, cncf sandbox, kubernetes]
---

# Introduction

Kubewarden is a Kubernetes Policy Engine.
It performs the tasks of a [Kubernetes Dynamic Admission Controller](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)
that validates incoming requests using WebAssembly policies.
Kubewarden also provides an audit scanner to actively check policy enforcement over time.

> You can develop policies in any programming language that generates WebAssembly binaries.

Reviewer: in the PE engine topic, is a fuzzy definition, more like a buzzword, but nevertheless it is one that users expect to mentally classify the project.

> You can develop policies in any programming language that generates WebAssembly binaries.

Author: Thanks, I made some changes.

keywords: [kubewarden, cncf, cncf sandbox, kubernetes]
---

# Introduction

Kubewarden is a Kubernetes Policy Engine.

> It aims to be the Universal Policy Engine for Kubernetes

Reviewer 2: ```suggestion
It aims to be the Universal Policy Engine for Kubernetes.
```

Author: Thanks.

> You can reuse policies from other policy engines without writing any code.

Reviewer 2: ```suggestion
You can reuse policies from other policy engines without having to rewrite them.
```
